source/core/parameters.cc

```diff
@@ -1548,6 +1558,39 @@ namespace Parameters
                           "velocity|pressure|phase|temperature"),
                         "Variable for kelly estimation"
                         "Choices are <velocity|pressure|phase|temperature>.");
+      // end WIP
+      // WIP declare multivariable parameters
+      prm.declare_entry(
+        "types",
+        "none",
+        Patterns::Anything(),
```

Comment:
It is not a good idea to use Patterns::Anything. There is a deal.II pattern for this, we already use it:
https://github.com/lethe-cfd/lethe/blob/master/source/core/parameters_lagrangian.cc#L313

Reply:
I was searching for that, thank you! Just a note that these `list x`, `y` and `z` are missing from the documentation (https://lethe-cfd.github.io/lethe/parameters/dem/insertion_info.html)